maintainers/maintainer-list.nix

```diff
@@ -1066,6 +1066,12 @@
     githubId = 14111;
     name = "Brandon Dimcheff";
   };
+  beardhatcode = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add beardhatcode``.

Reply:
Commit 19b77553918b36b92e702186b84498555fe15bc8 has been reworded to have the subject `maintainers: add beardhatcode` (045e01aae6adaf512b370bd046c5f1ae7ba8909a).